frontend/src/scenes/project/Settings/index.tsx

```diff
@@ -21,8 +21,46 @@ import { commandPaletteLogic } from 'lib/components/CommandPalette/commandPalett
 import { userLogic } from 'scenes/userLogic'
 import { JSBookmarklet } from 'lib/components/JSBookmarklet'
 
-export const Setup = hot(_Setup)
-function _Setup(): JSX.Element {
+function DisplayName(): JSX.Element {
+    const { currentTeam, currentTeamLoading } = useValues(teamLogic)
+    const { patchCurrentTeam } = useActions(teamLogic)
+
+    const [name, setName] = useState(currentTeam?.name || '')
+
+    if (currentTeam?.is_demo) {
+        return (
+            <p>
+                <i>The demo project cannot be renamed.</i>
+            </p>
+        )
+    }
+
+    return (
+        <div>
+            <Input
```

Comment:
From a UX perspective this "made me think" (took me a few seconds to understand I was renaming the project). A few options, have the input disabled and then add a button that says "Rename project" that enables the input. Add a helper text that explains what you're doing (with the "rename" word right in your face). Change the button copy to "Rename Project".

It would also be very helpful for future reference (and people without context on this particular PR) to include a screenshot of this in the PR description.

Reply:
Definitely, sorry for no screenshots. I specifically wrote "*If this affects the frontend, include screenshots.*" in the PR template but didn't adhere that. 😅

Reply:
Haha no worries, it was pretty small change too.